client/battle/BattleAnimationClasses.cpp

```diff
@@ -962,13 +962,13 @@ bool EffectAnimation::init()
 		}
 		else
 		{
-			const CStack * destStack = owner.getCurrentPlayerInterface()->cb->battleGetStackByPos(battlehexes[i], false);
+			const auto * destStack = owner.getCurrentPlayerInterface()->cb->battleGetUnitByPos(battlehexes[i], false);
 			Rect tilePos = owner.fieldController->hexPositionLocal(battlehexes[i]);
 
 			be.pos.x = tilePos.x + tilePos.w/2 - first->width()/2;
 
 			if(destStack && destStack->doubleWide()) // Correction for 2-hex creatures.
-				be.pos.x += (destStack->side == BattleSide::ATTACKER ? -1 : 1)*tilePos.w/2;
+				be.pos.x += (destStack->unitSide() == BattleSide::ATTACKER ? -1 : 1)*tilePos.w/2;

```

Comment:
What's the type of `destStack->unitSide()` and `destStack->side`? Last time I checked it was uint8_t or something. Feel free to change to `BattleSide` (and convert it to enum class if not already)

Reply:
@IvanSavenko later, firstly I want to push refactor + native terrain rework + this spell fixes.